bin/util.py

```diff
@@ -623,15 +723,12 @@ def write_yaml(data, path=None, allow_yamllib=False):
             return yamllib.dump(data)
         with open(path, "w") as stream:
             yamllib.dump(data, stream)
-        return
+        return None
```

Comment:
I think I prefer a blank `return` over `return None` :thinking: Ruff also has a rule for this, not sure why it's not triggered here: https://docs.astral.sh/ruff/rules/unnecessary-return-none/

Reply:
https://github.com/python/mypy/issues/13203